initial-promises/node-server/promises.cf

```diff
@@ -365,30 +365,44 @@ bundle agent check_uuid
 }
 
 #######################################################
+# Match all variants of cron proccesses
+
+body process_select cron_bin
+{
+  command => "(/usr/sbin/)?crond?( -n| -f)?";
```

Comment:
This allows matching the complete COMMAND, and not only the end, allowing not to match anacron for example.